src/solace_ai_connector/common/log.py

```diff
@@ -20,12 +42,15 @@ def setup_log(logFilePath, stdOutLogLevel, fileLogLevel):
 
     # Create an empty file at logFilePath (this will overwrite any existing content)
     with open(logFilePath, "w") as file:
-        file.write("")        
+        file.write("")
 
     # file_handler = logging.handlers.TimedRotatingFileHandler(
     #    filename=logFilePath, when='midnight', backupCount=30, mode='w')
     file_handler = logging.FileHandler(filename=logFilePath, mode="a")
-    file_formatter = logging.Formatter("%(asctime)s |  %(levelname)s: %(message)s")
+    if logFormat == "json":
```

Comment:
[Q] Wouldn't this be `jsonl` ?

Reply:
jsonl is a better option. Updated to the jsonl format. Thanks.